addons/mass_mailing/wizard/mail_compose_message.py

```diff
@@ -57,6 +57,7 @@ def get_mail_values(self, res_ids):
 
             blacklist = self._context.get('mass_mailing_blacklist')
             seen_list = self._context.get('mass_mailing_seen_list')
+            mass_mail_layout = self.env.ref('mass_mailing.mass_mailing_mail_layout')
```

Comment:
Need to add:
raise_if_not_found= False

(In order not to have any error and to send the mail (to take the same direction as the commit: https://github.com/odoo/odoo/commit/f8031a61f3885d2279e6490258e181dbb8d3130c )

Reply:
@Gorash  :+1:  done